### ISSUE TITLE
Initialisation fix r5954 breaks backward compatibility.

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#971]
**Reported by dietmarw on 23 Jan 2013 12:04 UTC**
In a recent change 2f8fc83b57417b7bd27bd92f69d6740d603f367f two public parameters were removed (commented out) from the model:
```mo
// parameter Modelica.SIunits.Voltage IC=0 "Initial Value"; 
// parameter Boolean UIC = false;
```

This change will break any existing user model using an inductor or capacitor that does supply modifiers to those parameters. Hence this fix can not be applied as such but must be corrected in a backward compatible way.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/971

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 23 Jan 2013 12:18 UTC**
This is a misunderstanding. In MSL 3.2, the two public parameters are **not **present. In MSL trunk, they have been introduced temporarily and then it was pointed out that this type of initialization is not as it is done in the rest of MSL. As a result, the two parameters have been removed and the initialization is defined via a start variable directly.

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 23 Jan 2013 12:22 UTC**
I see. BTW it helps _everybody_ to document those discussions in Trac rather than in inaccessible email references ;-)

**Comment by modelica-trac-importer** [comment]
**Modified by dietmarw on 23 Jan 2013 12:23 UTC**

**Comment by modelica-trac-importer** [comment]
**Comment by sjoelund&#46;se on 23 Jan 2013 12:23 UTC**
But why was the code commented out instead of removed? Subversion will remember the code was there at one point in time, but should users who look at the released code see it?

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 23 Jan 2013 12:25 UTC**
Good point. That really should be removed!

**Comment by modelica-trac-importer** [comment]
**Comment by clauss on 24 Jan 2013 09:31 UTC**
Fixed with 79f5e078a534f2bffed85a580a84517a656ae459: code removed.
